reana-ui/src/pages/workflowList/components/Welcome.js

```diff
@@ -64,12 +57,64 @@ export default function Welcome() {
       </p>
     </div>
   );
+}
 
+function WelcomeRegular() {
   return (
-    <Container text className={styles["container"]}>
-      <Title as="h2">Welcome to REANA!</Title>
-      {reanaToken ? tokenContent : <WelcomeNoTokenMsg />}
-    </Container>
+    <>
+      <p>
+        It seems that you are using REANA for the first time. Would you like to
+        try out a small example? Install the REANA command-line client
```

Comment:
We cannot say "Install the ..." here, because the shown example does more than that, i.e. it installs the client, sets up environment, clones example, and runs it.

I'd suggest to simply say:

```
It seems that you are using REANA for the first time. Would you 
like to try out a small example? Please proceed as follows:
```